js/src/components/contact-information/phone-number-card/verify-phone-number-content.js

```diff
@@ -0,0 +1,236 @@
+	return new Promise( ( resolve ) =>
+		setTimeout( () => resolve( `${ method }-12345` ), 1000 )
+	);
+};
+
+// TODO: [full-contact-info] remove mock functions
+const verifyPhoneNumber = ( id, code, method ) => {
+	// eslint-disable-next-line no-console
+	console.log( 'verifyPhoneNumber', { id, code, method } );
+	return new Promise( ( resolve, reject ) => {
+		setTimeout( () => {
+			if ( code === '000000' ) {
+				resolve();
+			} else {
+				reject( 'Incorrect verification code. Please try again.' );
+			}
+		}, 2000 );
+	} );
+};
+
+const appearanceDict = {
+	[ VERIFICATION_METHOD_SMS ]: {
+		toInstruction( phoneNumber ) {
+			return createInterpolateElement(
+				__(
+					'A text message with the 6-digit verification code has been sent to <userPhoneNumber />.',
+					'google-listings-and-ads'
+				),
+				{ userPhoneNumber: <strong>{ phoneNumber }</strong> }
+			);
+		},
+		textResend: __( 'Resend code', 'google-listings-and-ads' ),
+		// translators: %d: seconds to wait until the next verification code can be requested via SMS.
+		textResendCooldown: __(
+			'Resend code (in %ds)',
+			'google-listings-and-ads'
+		),
+		textSwitch: __(
+			'Or, receive a verification code through a phone call',
+			'google-listings-and-ads'
+		),
+	},
+	[ VERIFICATION_METHOD_PHONE_CALL ]: {
+		toInstruction( phoneNumber ) {
+			return createInterpolateElement(
+				__(
+					'You will receive a phone call at <userPhoneNumber /> with an automated message containing the 6-digit verification code.',
+					'google-listings-and-ads'
+				),
+				{ userPhoneNumber: <strong>{ phoneNumber }</strong> }
+			);
+		},
+		textResend: __( 'Call again', 'google-listings-and-ads' ),
+		// translators: %d: seconds to wait until the next verification code can be requested via phone call.
+		textResendCooldown: __(
+			'Call again (in %ds)',
+			'google-listings-and-ads'
+		),
+		textSwitch: __(
+			'Or, receive a verification code through text message',
+			'google-listings-and-ads'
+		),
+	},
+};
+
+/**
+ * Renders inputs for verifying phone number in Card.Body UI.
+ * Please note that this component will send a verification code request if the current method hasn't been requested yet after rendering.
+ *
+ * @param {Object} props React props.
+ * @param {string} props.verificationMethod The initial verification method.
+ * @param {string} props.country The country code. Example: 'US'.
+ * @param {string} props.number The phone number string in E.164 format. Example: '+12133734253'.
+ * @param {string} props.display The phone number string in international format. Example: '+1 213 373 4253'.
+ * @param {Function} props.onPhoneNumberVerified Called when the phone number is verified.
+ */
+export default function VerifyPhoneNumberContent( {
+	verificationMethod,
+	country,
+	number,
+	display,
+	onPhoneNumberVerified,
+} ) {
+	const [ method, setMethod ] = useState( verificationMethod );
+	const { second, callCount, startCountdown } = useCountdown( method );
```

Comment:
💅  Related to my other comment, if we make `useCountdown` deal with one timer, then we don't need to pass in `method`, and we can have something like this:

```js
	const smsCountdown = useCountdown();
	const phoneCountdown = useCountdown();
```

We can do this because there are only two fixed countdowns, the number of countdowns is not dynamic.